docs/src/examples/components/Dropdown/Types/DropdownExampleMultiple.shorthand.steps.ts

```diff
@@ -5,7 +5,7 @@ const selectors = {
   item: (itemIndex: number) => `.${Dropdown.slotClassNames.itemsList} li:nth-child(${itemIndex})`,
   removeItemIcon: (itemIndex: number) =>
     `.${Dropdown.slotClassNames.selectedItems} span:nth-child(${itemIndex}) .${
-      DropdownSelectedItem.slotClassNames.removeIcon
+      DropdownSelectedItem.slotClassNames.icon
```

Comment:
Please add **BREAKING CHANGE** section in the changelog, as you renamed the slot class name icon and in the PR description add steps of how the breaking changes should be resolved (mainly just replace the usages of  `DropdownSelectedItem.slotClassNames.removeIcon` with `DropdownSelectedItem.slotClassNames.icon`